src/components/modules/epics/ContributionsEpic.stories.tsx

```diff
@@ -3,11 +3,38 @@ import { Story, Meta } from '@storybook/react';
 import { ContributionsEpic, EpicProps } from './ContributionsEpic';
 import { TickerCountType, TickerEndType } from '../../../lib/variants';
 import { props } from './utils/storybook';
+import { from } from '@guardian/src-foundations/mq';
+import { css } from '@emotion/core';
+import { palette } from '@guardian/src-foundations';
+
+const containerStyles = css`
+    margin: 3em auto;
+    padding: 0 10px;
+    max-width: 620px;
+
+    ${from.mobileLandscape} {
+        padding: 0 20px;
+    }
+`;
+
+const backgroundStyles = css`
+    background-color: ${palette.neutral[97]};
+`;
+
+export const EpicDecorator = (Story: Story): JSX.Element => (
+    <div css={containerStyles}>
+        <div css={backgroundStyles}>
+            <Story />
+        </div>
+    </div>
+);
```

Comment:
This just makes the storybook version of the epic look more like it will on dotcom